src/pkg/podman/podman.go

```diff
@@ -27,8 +27,18 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type label struct {
+	Key   string
+	Value string
+}
+
 var (
-	LogLevel = logrus.ErrorLevel
+	LogLevel      = logrus.ErrorLevel
+	ToolboxLabels = []label{
+		{"com.github.debarshiray.toolbox", "true"},
+		{"com.github.containers.toolbox", "true"},
+		{"com.redhat.component", "fedora-toolbox"},
```

Comment:
I wonder if we are at a point where we can drop the `com.redhat.component` label. We need to check if there are still any supported `fedora-toolbox` images, and the possibility of creating containers, with only this label and none of the others. Digging into the Git history of the labels can help answer this question.

Reply:
I guess instead of just dropping the label we could have two lists: one for creating and the other for using in containers??

Reply:
Based on your comment in de5e5df9b793edc746cde7125cd437b69f15ccd7 it should be ok to drop the `com.redhat.component` label.